### DISCUSSION TITLE
旁路模式后台面板只显示ip

### DISCUSSION THREAD

**Discussion by XMethues** [nikkinikki-org/OpenWrt-nikki#66] [Q&A] [answered]
电脑网关和dns都指向旁路ip. 后台界面全是ip，分流失效。请问是哪里设置有问题吗？
![image](https://github.com/user-attachments/assets/a3a7a181-d1d5-4e7a-9fc3-f8ac5aa6e0a9)

**Comment by morytyann** [comment]
我的测试环境就是旁路由，应该兼容的呀，贴一下你的插件配置看看吧。

**Comment by morytyann** [comment]
😂是插件的配置，不是配置文件，打开 服务 -> Mihomo 截个图就行

**Reply by XMethues** [reply]
![image](https://github.com/user-attachments/assets/a7f46877-4021-444f-98aa-ad80d892a431)

**Comment by morytyann** [comment]
nslookup google.com 看一下返回？
面板只有IP，应该是DNS劫持没生效，也有可能是你开了访问控制，但没有添加对应的设备？
来自路由器的连接也是只有IP吗？

**Reply by XMethues** [reply]
访问控制是全部模式。看样子是没劫持到？
```
root@ImmortalWrt:~# nslookup google.com
Server:         127.0.0.1
Address:        127.0.0.1:53

Non-authoritative answer:
Name:   google.com
Address: 172.217.160.78

Non-authoritative answer:
Name:   google.com
Address: 2404:6800:4012:2::200e
```
![image](https://github.com/user-attachments/assets/eb703166-fbf8-4afe-ae09-494274d7a7b6)

**Reply by morytyann** [reply]
是的，`nft list table inet mihomo` 看看，可能是服务启动的时候哪里报错导致没能正常运行nftables脚本

**Reply by morytyann** [reply]
看截图好像没问题呢，你看看`dns_hijack_nfproto`这个集合有没有元素

**Reply by morytyann** [reply]
哦我知道了，你是ImmortalWrt，有一个DNSMASQ的选项需要关掉

**Reply by morytyann** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/99999844-8e83-4b6a-aaa9-5b460cb02693)这里的DNS 重定向，关掉它就好了

**Reply by XMethues** [reply]
Ok了。谢谢！！！！！
但是好像有缓存，有没有清理缓存的地方？

**Reply by morytyann** [reply]
电脑的DNS缓存吗？CMD运行`ipconfig /flushdns`或者PowerShell运行`Clear-DnsClientCache`即可。
移动设备的话一般开关飞行模式就可以了。
